docs/accounts/fxa-client-android.md

```diff
@@ -0,0 +1,122 @@
+---
+id: account-client-android
+title: Android SDK
+---
+
+The **Firefox Accounts Android SDK** provides a way for Android applications to do the following:
```

Comment:
let's mirror this on https://github.com/mozilla-mobile/android-components/tree/master/components/service/firefox-accounts#setting-up-the-dependency

Reply:
the sample-fxa app can describe some of the functionality, custom tab, vs webview, vs wantsKeys...